pipeline-utils.groovy

```diff
@@ -139,6 +139,27 @@ def sh_capture(cmd) {
     return sh(returnStdout: true, script: cmd).trim()
 }
 
+def registry_login(username, password, registry) {
+    sh "podman login -u '${username}' -p '${password}' ${registry}"
```

Comment:
Note: I believe this will populate `${XDG_RUNTIME_DIR}/containers/auth.json` (EG: `/run/user/1000/containers/auth.json`). Let's make sure this file doesn't get shown via the output or links.

Reply:
That *should* be outside the `$WORKSPACE`, so the JSON shouldn't get archived anywhere.  And there's no output from the `podman login` except `Login Succeeded!`

For example, here - https://github.com/openshift/os/blob/1143fc94b8d9f70dcace9d3845053fc5557c57e0/scripts/pull-mount-oscontainer#L10

...looks like this from the pipeline logs

```
+ set +x
podman login -u unused -p <password> registry.svc.ci.openshift.org
Login Succeeded!
```